firmware/nrf52/Cargo.toml

```diff
@@ -15,7 +15,7 @@ cortex-m-rt = "0.6.10"
 cortex-m-semihosting = "0.3.3"
 embedded-hal = "0.2.4"
 nrf52840-hal = "0.10.0"
-panic-probe = { path = "../panic-probe" }
+panic-probe = { git = "https://github.com/knurling-rs/probe-run", branch = "main" }
```

Comment:
Should we pin this / publish a new panic-probe version first?

Reply:
hmm, yeah. I think we should pin to a commit and then move this to a crates.io version.

Reply:
(we don't actually run these apps so we won't hit panics but it's best to pin the dependency)

Reply:
Yeah, I was thinking of deleting the nrf52-specific stuff for that reason. But it doesn't hurt to keep it I guess. I think we do build them in CI though.